docs/interpreter/hive.md

```diff
@@ -9,93 +9,51 @@ group: manual
-  </tr>
-  <tr>
-    <td>default.url</td>
-    <td>jdbc:hive2://localhost:10000</td>
-    <td>Url for connection</td>
-  </tr>
-  <tr>
-    <td>default.user</td>
-    <td></td>
-    <td><b>( Optional ) </b>Username of the connection</td>
-  </tr>
-  <tr>
-    <td>default.password</td>
-    <td></td>
-    <td><b>( Optional ) </b>Password of the connection</td>
-  </tr>
-  <tr>
-    <td>default.xxx</td>
-    <td></td>
-    <td><b>( Optional ) </b>Other properties used by the driver</td>
-  </tr>
-  <tr>
-    <td>${prefix}.driver</td>
-    <td></td>
-    <td>Driver class path of <code>%hive(${prefix})</code> </td>
-  </tr>
-  <tr>
-    <td>${prefix}.url</td>
-    <td></td>
-    <td>Url of <code>%hive(${prefix})</code> </td>
-  </tr>
-  <tr>
-    <td>${prefix}.user</td>
-    <td></td>
-    <td><b>( Optional ) </b>Username of the connection of <code>%hive(${prefix})</code> </td>
-  </tr>
-  <tr>
-    <td>${prefix}.password</td>
-    <td></td>
-    <td><b>( Optional ) </b>Password of the connection of <code>%hive(${prefix})</code> </td>
-  </tr>
-  <tr>
-    <td>${prefix}.xxx</td>
-    <td></td>
-    <td><b>( Optional ) </b>Other properties used by the driver of <code>%hive(${prefix})</code> </td>
-  </tr>
-</table>
-
-This interpreter provides multiple configuration with `${prefix}`. User can set a multiple connection properties by this prefix. It can be used like `%hive(${prefix})`.
+Hive Interpreter is merged into JDBC Interpreter. You can see JDBC Interpreter [here](./jdbc.html)
+
+## Example
+### Properties
+ <table class="table-configuration">
+   <tr>
+     <th>Name</th>
+     <th>Value</th>
+   </tr>
+   <tr>
+     <td>hive.driver</td>
+     <td>org.apache.hive.jdbc.HiveDriver</td>
+   </tr>
+   <tr>
+     <td>hive.url</td>
+     <td>jdbc:hive2://localhost:10000</td>
+   </tr>
+   <tr>
+     <td>hive.user</td>
+     <td>hive_user</td>
+   </tr>
+   <tr>
+     <td>hive.password</td>
+     <td>hive_password</td>
+   </tr>
+ </table>
+### Dependencies
+ <table class="table-configuration">
+   <tr>
+     <th>Artifact</th>
+     <th>Excludes</th>
+   </tr>
+   <tr>
+     <td>org.apache.hive:hive-jdbc:0.14.0</td>
+     <td></td>
+   </tr>
+   <tr>
+     <td>org.apache.hadoop:hadoop-common:2.6.0</td>
+     <td></td>
+   </tr>
+ </table>
 
 ## How to use
-Basically, you can use
-
-```sql
-%hive
-select * from my_table;
-```
-
-or
-
-```sql
-%hive(etl)
--- 'etl' is a ${prefix}
-select * from my_table;
+ 
 ```
-
-You can also run multiple queries up to 10 by default. Changing these settings is not implemented yet.
-
-### Apply Zeppelin Dynamic Forms
-You can leverage [Zeppelin Dynamic Form]({{BASE_PATH}}/manual/dynamicform.html) inside your queries. You can use both the `text input` and `select form` parameterization features.
-
-```sql
-%hive
-SELECT ${group_by}, count(*) as count
-FROM retail_demo.order_lineitems_pxf
-GROUP BY ${group_by=product_id,product_id|product_name|customer_id|store_id}
-ORDER BY count ${order=DESC,DESC|ASC}
-LIMIT ${limit=10};
```

Comment:
@bzz @felixcheung OK. I'll revert hive.md and add some notification about how to move hive to jdbc. I also think it would be better and less confusion to existing users.